src/client/client.ts

```diff
@@ -281,12 +428,22 @@ export default class CTProtoClient<AuthRequestPayload, AuthResponsePayload, ApiR
       const message = JSON.parse(event.data.toString());
       const messageId = message.messageId;
 
-      if ('type' in message) {
+      const payload = message.payload;
```

Comment:
```suggestion
      const payload = message.payload;

```

andf other code should be linted